>From 0721bc8cdb3d4e22c7e2358df5fd0b87a50fa07e Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sat, 31 May 2014 23:33:21 +0900
Subject: [PATCH] fix lintian "vcs-field-not-canonical" warning

use anonscm.debian.org, instead of git.debian.org
---
 lib/debiani/control | 4 ++--
 lib/debiank/control | 4 ++--
 lib/debianl/control | 4 ++--
 lib/debianm/control | 4 ++--
 lib/debiann/control | 4 ++--
 lib/debians/control | 4 ++--
 6 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/lib/debiani/control b/lib/debiani/control
index d935e63..78d8203 100644
--- a/lib/debiani/control
+++ b/lib/debiani/control
@@ -5,8 +5,8 @@ Maintainer: #USERNAME# <#EMAIL#>
 Build-Depends: #BUILD_DEPS#
 Standards-Version: #POLICY#
 Homepage: <insert the upstream URL, if relevant>
-#Vcs-Git: git://git.debian.org/collab-maint/#PACKAGE#.git
-#Vcs-Browser: http://git.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
+#Vcs-Git: git://anonscm.debian.org/collab-maint/#PACKAGE#.git
+#Vcs-Browser: http://anonscm.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
 
 Package: #PACKAGE#
 Architecture: all
diff --git a/lib/debiank/control b/lib/debiank/control
index dda7a7d..aca84da 100644
--- a/lib/debiank/control
+++ b/lib/debiank/control
@@ -5,8 +5,8 @@ Maintainer: #USERNAME# <#EMAIL#>
 Build-Depends: #BUILD_DEPS#
 Standards-Version: #POLICY#
 Homepage: <insert the upstream URL, if relevant>
-#Vcs-Git: git://git.debian.org/collab-maint/#PACKAGE#.git
-#Vcs-Browser: http://git.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
+#Vcs-Git: git://anonscm.debian.org/collab-maint/#PACKAGE#.git
+#Vcs-Browser: http://anonscm.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
 
 Package: #PACKAGE#-utils
 Architecture: any
diff --git a/lib/debianl/control b/lib/debianl/control
index d7db427..458606a 100644
--- a/lib/debianl/control
+++ b/lib/debianl/control
@@ -5,8 +5,8 @@ Build-Depends: #BUILD_DEPS#
 Standards-Version: #POLICY#
 Section: libs
 Homepage: <insert the upstream URL, if relevant>
-#Vcs-Git: git://git.debian.org/collab-maint/#PACKAGE#.git
-#Vcs-Browser: http://git.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
+#Vcs-Git: git://anonscm.debian.org/collab-maint/#PACKAGE#.git
+#Vcs-Browser: http://anonscm.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
 
 Package: #PACKAGE#-dev
 Section: libdevel
diff --git a/lib/debianm/control b/lib/debianm/control
index 846978a..71d2134 100644
--- a/lib/debianm/control
+++ b/lib/debianm/control
@@ -5,8 +5,8 @@ Maintainer: #USERNAME# <#EMAIL#>
 Build-Depends: #BUILD_DEPS#
 Standards-Version: #POLICY#
 Homepage: <insert the upstream URL, if relevant>
-#Vcs-Git: git://git.debian.org/collab-maint/#PACKAGE#.git
-#Vcs-Browser: http://git.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
+#Vcs-Git: git://anonscm.debian.org/collab-maint/#PACKAGE#.git
+#Vcs-Browser: http://anonscm.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
 
 Package: #PACKAGE#
 Architecture: any
diff --git a/lib/debiann/control b/lib/debiann/control
index 5dbd959..703ae01 100644
--- a/lib/debiann/control
+++ b/lib/debiann/control
@@ -5,8 +5,8 @@ Maintainer: #USERNAME# <#EMAIL#>
 Build-Depends: #BUILD_DEPS#
 Standards-Version: #POLICY#
 Homepage: <insert the upstream URL, if relevant>
-#Vcs-Git: git://git.debian.org/collab-maint/#PACKAGE#.git
-#Vcs-Browser: http://git.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
+#Vcs-Git: git://anonscm.debian.org/collab-maint/#PACKAGE#.git
+#Vcs-Browser: http://anonscm.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
 
 Package: #PACKAGE#
 Architecture: all
diff --git a/lib/debians/control b/lib/debians/control
index 24c694f..e9c1f6d 100644
--- a/lib/debians/control
+++ b/lib/debians/control
@@ -5,8 +5,8 @@ Maintainer: #USERNAME# <#EMAIL#>
 Build-Depends: #BUILD_DEPS#
 Standards-Version: #POLICY#
 Homepage: <insert the upstream URL, if relevant>
-#Vcs-Git: git://git.debian.org/collab-maint/#PACKAGE#.git
-#Vcs-Browser: http://git.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
+#Vcs-Git: git://anonscm.debian.org/collab-maint/#PACKAGE#.git
+#Vcs-Browser: http://anonscm.debian.org/?p=collab-maint/#PACKAGE#.git;a=summary
 
 Package: #PACKAGE#
 Architecture: any
-- 
2.0.0

